second_test/second_test.spec.js

```diff
@@ -0,0 +1,37 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test('Perform login', async ({ page }) => {
+	await page.goto('https://www.skroutz.gr/');
+	//await page.getByRole('link', { name: 'Σύνδεση / Εγγραφή' }).click();
+	await page.click('a.icon');
+	await page.getByPlaceholder('Email').click();
+	await page.getByPlaceholder('Email').fill('polegar85@gmail.com');
+	//await page.getByRole('button', { name: 'Συνέχεια', exact: true }).click();
+	await page.click('button.btn-signin.check-email');
+	
+	await page.getByPlaceholder('Συνθηματικό').click();
```

Comment:
Once more, please stick with IDs if possible (`password-input-user[password]`).

Reply:
@sgkiokas I made a lot of effort with IDs and I can't find an alternative. could you help me some more please?